packages/callhierarchy/src/browser/callhierarchy-tree/callhierarchy-tree-widget.tsx

```diff
@@ -103,7 +103,15 @@ export class CallHierarchyTreeWidget extends TreeWidget {
         const symbol = definition.symbolName;
         const location = this.labelProvider.getName(new URI(definition.location.uri));
         const container = (containerName) ? containerName + ' — ' + location : location;
-        return <div className='definitionNode'>
+        const isDeprecated = definition.tags?.includes(SymbolTag.Deprecated);
+        const classNames: string[] = ['definitionNode'];
```

Comment:
We can let TypeScript infer here:

```suggestion
        const classNames = ['definitionNode'];
```